UK/Thames/Southend APP.prf

```diff
@@ -1,23 +1,31 @@
 Settings	SettingsfilePLUGINS	\..\Data\Settings\Local\Thames\Southend_Plugins.txt
-Settings	SettingsfilePROFILE	\..\Data\Settings\Local\Thames\Profiles.txt
+Settings	SettingsfilePROFILE	\..\Data\Settings\Local\Southend\Profiles.txt
 Settings	SettingsfileSCREEN	\..\Data\Settings\TC_APP\TC_APP_Screen.txt
 Settings	SettingsfileVOICE	\..\Data\Settings\Voice.txt
 Settings	Settingsfile	\..\Data\Settings\TC_APP\TC_APP_General.txt
-Settings	SettingsfileARR	\..\Data\Settings\TC_APP\TC_APP_AL_SMR.txt
+Settings	SettingsfileARR	\..\Data\Settings\TC_APP\TC_APP_AL.txt
 Settings	SettingsfileCONFLICT	\..\Data\Settings\TC_APP\TC_APP_C.txt
 Settings	SettingsfileDEP	\..\Data\Settings\TC_APP\TC_APP_DL.txt
 Settings	SettingsfileFP	\..\Data\Settings\TC_APP\TC_APP_FPL.txt
-Settings	SettingsfileSEL	\..\Data\Settings\TC_APP\TC_APP_SEL_SMR.txt
-Settings	SettingsfileSIL	\..\Data\Settings\TC_APP\TC_APP_SIL_SMR.txt
-Settings	SettingsfileSYMBOLOGY	\..\Data\Settings\TC_APP\TC_APP_Symbology.txt
+Settings	SettingsfileSEL	\..\Data\Settings\TC_APP\TC_APP_SEL.txt
+Settings	SettingsfileSIL	\..\Data\Settings\TC_APP\TC_APP_SIL.txt
+Settings	SettingsfileSYMBOLOGY	\..\Data\Settings\Local\Southend\MC_APP_Symbology.txt
 Settings	SettingsfileTAGS	\..\Data\Settings\Tags.txt
 Settings	SettingsfileVCCS	\..\Data\Settings\VCCS_South.txt
-Settings	sector	UK\Data\Sector\UK_2023_03.sct
+Settings	sector	UK\Data\Sector\UK_2023_04.sct
```

Comment:
Could you leave at the same revision as the rest of the profiles? This will ensure I don't miss the update on release